src/components/DeploymentDetails.vue

```diff
@@ -137,6 +146,7 @@
 
   const can = useCan()
   const api = useWorkspaceApi()
+  const serviceLevelAgreements = computed(() => props.slas)
```

Comment:
Don't use a computed for this. You can just reference `slas` directly in the template.